ambry-account/src/main/java/com/github/ambry/account/HelixAccountService.java

```diff
@@ -155,14 +158,13 @@
     };
     updater.run();
     if (scheduler != null) {
-      int initialDelay = new Random().nextInt(storeConfig.accountUpdaterPollingIntervalMs + 1);
-      scheduler.scheduleAtFixedRate(updater, initialDelay, storeConfig.accountUpdaterPollingIntervalMs,
-          TimeUnit.MILLISECONDS);
+      int initialDelay = new Random().nextInt(config.updaterPollingIntervalMs + 1);
+      scheduler.scheduleAtFixedRate(updater, initialDelay, config.updaterPollingIntervalMs, TimeUnit.MILLISECONDS);
       logger.info(
           "Background account updater will fetch accounts from remote starting {} ms from now and repeat with interval={} ms",
-          initialDelay, storeConfig.accountUpdaterPollingIntervalMs);
+          initialDelay, config.updaterPollingIntervalMs);
     }
-    isOpen = true;
+    open.set(true);
```

Comment:
this is `true` already by virtue of the construction. Should this start off as `false` then?

Reply:
This line should be removed since I changed the bool to start off as true. There is no reason to start it as false because none of the methods can be accessed if the constructor call did not complete successfully.